packages/airnode-adapter/package.json

```diff
@@ -34,7 +34,7 @@
     "@types/mocha": "^10.0.1",
     "chai": "^4.3.7",
     "ethereum-waffle": "^4.0.10",
-    "hardhat": "2.10.2",
+    "hardhat": "2.14.0",
```

Comment:
Why is this a fixed version? (Renovate does not update it because of this). My guess is @bbenligiray was afraid about ABI changes?

Reply:
@Siegrift you actually pinned this in https://github.com/api3dao/airnode/pull/1325#issuecomment-1195090498  😅

Reply:
Yeah, I believe there was some issue with the combination of the plugins. I don't think this has any impact on the contracts so merging is fine (and also unpinning)